var/spack/repos/builtin/packages/tcl-bwidget/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class TclBwidget(Package):
```

Comment:
All of the current packages that extend `tcl` inherit from `AutotoolsPackage`.  Is this something to consider?

Also, several packages, including `tcl`, `tclap`, and `tk`, "inherit" from `SourceforgePackage` (actually a `mixin`) for the benefits of setting the url and mirrors.  

Have you considered at least taking advantage of the mixin?